ONSdigital/eq-survey-runner
# Able to return to ras from eq via 'My account' link in header

### What is the context of this PR?
https://trello.com/c/uTTLYYwX/2017-able-to-return-to-ras-from-eq

This adds the 'My account' link to the header of every page. 

This shouldn't be merged yet as I've not yet made the account_url mandatory and got the tests working at the same time.

### How to review 
- Open any survey, in the header should be 'My account' on every page throughout the survey.
- After launching a survey, add an extra slash and some garbage e.g., `/blah` to fire up an error page.  My account link should still appear if it was in the original payload when the survey was launched.
- Go to `localhost:5000` (or whatever port runner is on), an error page should appear WITHOUT a my account link in the header


### Checklist
